optimum/exporters/openvino/model_patcher.py

```diff
@@ -951,15 +951,103 @@ def __exit__(self, exc_type, exc_value, traceback):
                 block.attention.forward = block.attention._orig_forward
 
 
+# Adapted from Phi3Attention.forward
```

Comment:
would be very useful for  maintenance to also add the commit + exact file the original method is from, like for example :
```suggestion
# Adapted from https://github.com/huggingface/transformers/blob/ccdabc5642bf84849af93f591e207dc625c8e1e1/src/transformers/models/phi3/modeling_phi3.py#L426
```